checker/src/behavior/template_literal.rs

```diff
@@ -101,21 +101,23 @@ where
 		let call_site = position.clone().with_source(environment.get_source());
 		crate::types::calling::call_type_handle_errors(
 			tag,
-			crate::types::calling::CalledWithNew::None,
-			crate::behavior::functions::ThisValue::UseParent,
-			None,
-			arguments,
-			call_site,
+			CallingInput {
+				called_with_new: crate::types::calling::CalledWithNew::None,
+				this_value: crate::behavior::functions::ThisValue::UseParent,
+				call_site,
+				call_site_type_arguments: None,
+			},
 			environment,
+			arguments,
```

Comment:
There's a reason I didn't add it, it's because `fn call` in `checker/types/callings` takes `arguments: &[SynthesisedArgument]`.
So if we want to add `arguments` in `CallingInput`, it'll mean that we need another struct with `arguments` only for this function. Both are fine I think, do you have a preference?